### PR TITLE
Fix oauth2 mock

### PR DESCRIPTION
**Pull request by raymondfeng**
## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [x] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by raymondfeng** [comment]
Fixes https://github.com/strongloop/loopback-next/issues/5520

File: fixtures/mock-oauth2-provider/package.json [hunk 1]
```diff
@@ -47,5 +48,8 @@
   "devDependencies": {
     "@loopback/build": "^5.4.1",
     "@loopback/eslint-config": "^7.0.1"
+  },
+  "publishConfig": {
```
Comment (bajtos) [inline fixtures/mock-oauth2-provider/package.json:52]: Can you please move this toward the top of the file, for consistency with other package.json files?

File: fixtures/mock-oauth2-provider/package.json [hunk 2]
```diff
@@ -2,10 +2,11 @@
   "name": "@loopback/mock-oauth2-provider",
   "version": "0.0.3",
   "description": "mocks the oauth2 authorization flow",
-  "private": true,
```
Comment (bajtos) [inline fixtures/mock-oauth2-provider/package.json:5]: I thought we were intentionally keeping this package private ?

**Comment by bajtos** [comment]
See also https://github.com/strongloop/loopback-next/pull/5523 which is a smaller version of this PR.

File: fixtures/mock-oauth2-provider/package.json (hunk 2 above)
Comment (raymondfeng) [reply fixtures/mock-oauth2-provider/package.json:5]: Per #5520, when the `example-passport-login` is cloned with `lb4 example`, `npm i` fails as this package is devDep of `example-passport-login`.

**Pull request merged**